src/Plugin_Command.php

```diff
@@ -875,7 +904,7 @@ protected function get_item_list() {
 
 					if ( isset( $plugin_update_info->requires ) && version_compare( $wp_version, $requires, '>=' ) ) {
 						$reason = "This update requires WordPress version $plugin_update_info->requires, but the version installed is $wp_version.";
-					} elseif ( ! isset( $update_info['package'] ) ) {
+					} elseif ( ! isset( $plugin_update_info->package ) ) {
```

Comment:
`$update_info` does not exist at this point.

Introduced in #451

Reply:
🤦‍♀️ yes that is a typo since these are similarly named. I should set up PHPStan (!)